supporting_content/metrics/listunitprice.md

```diff
@@ -6,72 +6,79 @@ Current column mappings found in available data sets:
-  - related UC scenarios
-  - Current values observed in billing data for various scenarios
-  - Alternative data sources for various scenarios
-- See [Pricing Support – UCs and Data samples Spreadsheet](https://docs.google.com/spreadsheets/d/1AZ-vtkKeKwYc8rqhxP1zMTnAVAS-svmWQQmr8cpv-IM/edit#gid=117987709) for additional UC scenarios and sample data.
+* See [Appendix: Quantity-Driven Pricing strategies](../appendix/quantity_driven_pricing_strategies.md) section for more information on
+  * Quantity-Driven Pricing strategies (such as flat-rate, volume-based rates and tier-based rating)
+  * related UC scenarios
+  * Current values observed in billing data for various scenarios
+  * Alternative data sources for various scenarios
+* See [Pricing Support – UCs and Data samples Spreadsheet](https://docs.google.com/spreadsheets/d/1AZ-vtkKeKwYc8rqhxP1zMTnAVAS-svmWQQmr8cpv-IM/edit#gid=117987709) for additional UC scenarios and sample data.
 
 ## Discussion / Scratch space
 
+### Prices and Currencies
+
+* We deliberated on whether to specify both BillingCurrency and PricingCurrency and decided to enforce a single currency, specifically BillingCurrency. This approach provides consistency and simplifies invoice reconciliation. Since some providers don't include List Unit Prices in BillingCurrency in their public price sheets, the inclusion of a CurrencyExchangeRate dimension in the billing data becomes imperative (name subject to change). This is necessary to ensure we can accurately compare and match ListUnitPrices provided in billing data with those published in public price sheets.
+
+### Free-tier, volume/tier-based, BYOL-based and dynamically-priced SKU rates
+
+* Being inclusive of free-tier, volume/tier-based, BYOL-based and dynamically priced SKU rates, the List Unit Price cannot be used for calculating savings based on volume/tier-based pricing, the use of pre-owned software licenses (BYOL - Bring Your Own License), leveraging interruptible resources and/or services, or optimizing usage to take advantage of dynamic pricing models.
+
 ### Current data sources
 
-- While Microsoft and AWS offer candidate columns for mapping to ListUnitPrice within billing data, relying on pricing data for all providers is advisable due to mentioned discrepancies and volume/tier-based pricing concerns. Filtering pricing data based on resolved billing data to identify that specific list unit price is possible but often presents a challenge, particularly with volume/tier-based prices.
-- MSFT lists the Retail Price as well as the Unit Price.
+* For GCP and OCI, Pricing Data serves as the sole data source for ListUnitPrice, while for AWS, it is the preferred data source due to concerns related to volume/tier-based pricing. To determine the relevant price in Pricing Data for a specific charge record, the billing data must include the UnitPriceId. As previously mentioned, when BillingCurrency and PricingCurrency differ, the exchange rate must also be considered.
```

Comment:
> the billing data must include the UnitPriceId.

I understood the final name for UnitPriceId is actually SKUPriceId.

Reply:
Thank you for pointing this out! The note refers to a discussion that took place long before the SkuPriceId column was named and specified. An update is coming soon.

Reply:
supporting content updated (a bit more details provided).
resolving the conversation